wolfssl/certs_test.h

```diff
@@ -1471,6 +1471,228 @@ static const unsigned char ca_cert_der_2048[] =
 };
 static const int sizeof_ca_cert_der_2048 = sizeof(ca_cert_der_2048);
 
+static const unsigned char ca_cert_chain_der[] =
```

Comment:
This file is generated from the `./gencertbufs.pl`, so this new `ca_cert_chain_der` will be overwritten. For this to be easily maintained we need to add something in `./gencertbufs.pl` to combine two ca certs and output to `wolfssl/certs_test.h`.